gcc/multiple_target.cc

```diff
@@ -385,9 +386,11 @@ expand_target_clones (struct cgraph_node *node, bool definition)
   before = decl1_v;
   DECL_FUNCTION_VERSIONED (node->decl) = 1;
 
+  const int num_attrs = 2;
+  char attrs2[num_attrs][5] = {"sve","sve2"};
   for (i = 0; i < attrnum; i++)
     {
-      char *attr = attrs[i];
```

Comment:
This change seems like it would break attributes normally?
You're not parsing attrs[i] anymore and you're replacing it with your own.

Can you create a copy of the attrs char array and add the two extra strings for sve and sve2?